src/main/java/io/github/amerebagatelle/fabricskyboxes/skyboxes/textured/TexturedSkybox.java

```diff
@@ -1,21 +1,38 @@
 package io.github.amerebagatelle.fabricskyboxes.skyboxes.textured;
 
+import java.util.List;
+
+import com.google.common.collect.Lists;
 import com.mojang.blaze3d.platform.GlStateManager;
 import com.mojang.blaze3d.systems.RenderSystem;
 import io.github.amerebagatelle.fabricskyboxes.mixin.skybox.WorldRendererAccess;
 import io.github.amerebagatelle.fabricskyboxes.skyboxes.AbstractSkybox;
 import io.github.amerebagatelle.fabricskyboxes.util.JsonObjectWrapper;
+import io.github.amerebagatelle.fabricskyboxes.util.object.Fade;
+import io.github.amerebagatelle.fabricskyboxes.util.object.HeightEntry;
+import io.github.amerebagatelle.fabricskyboxes.util.object.RGBA;
+
 import net.minecraft.client.MinecraftClient;
 import net.minecraft.client.render.BufferBuilder;
 import net.minecraft.client.render.Tessellator;
 import net.minecraft.client.util.math.MatrixStack;
 import net.minecraft.client.util.math.Vector3f;
 import net.minecraft.client.world.ClientWorld;
+import net.minecraft.util.Identifier;
 
 public abstract class TexturedSkybox extends AbstractSkybox {
-    public float[] axis;
```

Comment:
Is there a reason why this needed to change?

Reply:
it's now a list array